drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/CollectionViewImpl.html

```diff
@@ -51,7 +51,7 @@ <h4 class="modal-title" data-field="editorTitle"><span class="fa fa-list"></span
                     </div>
                 </div>
             </div>
-            <div class="modal-body" data-field="addItemButtonContainer">
+            <div class="modal-body" style="height: 50px">
```

Comment:
Is it possible to use a CSS `class` instead of a fixed value?

Reply:
@danielezonca done.